numba/dppl_config.py

```diff
@@ -1,8 +1,5 @@
-dppl_present = False
-
 try:
     import dpctl
+    dppl_present = dpctl.has_gpu_queues() and dpctl.has_cpu_queues()
```

Comment:
OR

Reply:
Argh! Yes. You are right.

Reply:
I doubt numba.dppl works with CPU devices.
@reazulhoque can it work when only CPU devices available?

Reply:
The intent of the change is to make sure there is at least some OpenCL device (CPU or GPU) for Numba.dppl to use. We can also use dpctl.has_sycl_platforms() for this check.

Reply:
@PokhodenkoSA it should work with any sycl device.